queries.sql

```diff
@@ -9,6 +9,53 @@ SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE name NOT IN ('Gabumon');
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT delete;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO delete;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, COUNT(*) as escape_attempts_count
+FROM animals
+WHERE escape_attempts > 0
+GROUP BY neutered
+ORDER BY escape_attempts_count DESC;
+
```

Comment:
## Your Table :bookmark_tabs:
![image](https://user-images.githubusercontent.com/93039359/236152877-f06b3f03-2b82-4019-84a3-3873e2fc2a64.png)

### Your Result :red_circle:
![image](https://user-images.githubusercontent.com/93039359/236150960-748bdd97-5637-4fa7-b179-a492caeca94e.png)

### Expected Result :green_circle:
![image](https://user-images.githubusercontent.com/93039359/236151152-9004c87b-ba78-41b9-b831-72e1f0d3997a.png)

- [ ] It would be nice if you update this query to get the expected results which you can see in the image above :+1:

Reply:
![Screenshot 2023-05-04 164311](https://user-images.githubusercontent.com/44966918/236225876-399d860c-e20e-449f-b7ab-b639d0d30323.png)